src/test/java/bio/terra/app/controller/DatasetValidationsTest.java

```diff
@@ -121,6 +121,27 @@ public void testInvalidDatasetRequest() throws Exception {
             .andExpect(status().is4xxClientError());
     }
 
+    @Test
+    public void testJsonParsingErrors() throws Exception {
+        String invalidSchema = "{\"name\":\"no_response\"," +
```

Comment:
You can get rid of the date partition options, partition mode, and primaryKey fields to simplify this example. And perhaps use a more fake-sounding field name so that it's easy to tell what's expected?